src/components/loginSignup/login.jsx

```diff
@@ -18,9 +18,37 @@ export default class Login extends Component {
 
   handleInput = (stateName) => (e) => {
     e.preventDefault()
+    this.setState({
+      [stateName]: e.target.value.trim(),
+      errors: {
+        username: '',
+        password: '',
+      }
```

Comment:
This is not necessary if `handleError` is being called

Reply:
thats by mistake..i didnt see that